hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -196,22 +196,22 @@ def from_dict(cls, d):
 
 
 @dataclass
-class MatchesForHashResponse(JSONifiable):
-    matches: t.List[ThreatExchangeSignalMetadata]
-    signal_value: str
+class MatchesForHash(JSONifiable):
+    match_distance: int
+    matched_signal_fields: ThreatExchangeSignalMetadata  # or matches signal from other sources
```

Comment:
struggling to understand the name here. Why are these matched signal fields?

`matched_signals` instead?

Reply:
fair!

Reply:
`matched_signal` because it is a single matched signal